src/views/Ifos/components/Hero.tsx

```diff
@@ -1,29 +1,47 @@
 import React from 'react'
 import styled from 'styled-components'
-import { Heading, Text } from '@pancakeswap-libs/uikit'
+import { Box, Heading, Text } from '@pancakeswap-libs/uikit'
 import Container from 'components/layout/Container'
 import useI18n from 'hooks/useI18n'
 
+const getGradient = (isDark: boolean) => {
+  if (isDark) {
+    return 'repeating-linear-gradient(to right, #332453, #332453 40px, #281D44 40px, #281D44 80px)'
+  }
+
+  return 'repeating-linear-gradient(to right, #21d4e2, #21d4e2 40px, #53dee9 40px, #53dee9 80px)'
+}
+
```

Comment:
The reason I chose to use direct values is because `#21d4e2` is not in the current color palette. If this ribbon pattern starts to be re-used then I suggest we put it in the UI Kit.